spec/models/importers/organisation_spec.rb

```diff
@@ -1,19 +1,40 @@
 require 'rails_helper'
```

Comment:
There are a lot of things hapening in this commit.
Could you please extract separate commits for your refactorings. It is going to be much easier to review.